javascript-timers/javascript-timers-quiz-notes.md

```diff
@@ -0,0 +1,45 @@
+# javascript-timers-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is a "callback" function?
+  A callback function is a function passed into another function as an argument, which is then invoked inside the outer function to complete some kind of routine or action.
+- Besides adding an event listener callback function to an element or the `document`, what is one way to delay the execution of a JavaScript function until some point in the future?
+  setTimeout method
+- How can you set up a function to be called repeatedly without using a loop?
+
+- What is the default time delay if you omit the `delay` parameter from `setTimeout()` or `setInterval()`?
+  0
+- What do `setTimeout()` and `setInterval()` return?
+  setTimeout(function, milliseconds ) Executes a function, after waiting a specified number of milliseconds.
+  setInterval(function, milliseconds ) Same as setTimeout(), but repeats the execution of the function continuously.
```

Comment:
This doesn't explain what they actually return

Reply:
updated following quiz questions